x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_pause_provider.js

```diff
@@ -56,11 +56,11 @@ class Provider extends PureComponent {
     const title = isSingle
       ? intl.formatMessage({
         id: 'xpack.crossClusterReplication.pauseFollowerIndex.confirmModal.pauseSingleTitle',
-        defaultMessage: 'Pause follower index \'{name}\'?',
+        defaultMessage: 'Pause data replication of follower index \'{name}\'?',
```

Comment:
I have some concerns about using the phrase `data replication` here and elsewhere because I think it's not entirely accurate, CCR replicates any type of operations from leader, and those operations could be data related or non-data related. what do you think of omitting `data`, e.g. `Pause replication of follower index 'abc'?`

Reply:
Good suggestion! I think this makes sense.